api/api/spec/spec.yaml

```diff
@@ -2888,8 +3035,8 @@ tags:
   - name: syscollector
     description: 'Capability to collect interesting information for each agent'
 
-security:
-    - jwt: []
+#security:
+#    - jwt: []
```

Comment:
Why is this commented?

Reply:
Sorry I forgot removing the slashes. It's my trick to do testing without having to authenticate ;)